rollup-standalone.config.js

```diff
@@ -1,8 +1,9 @@
 import replace from '@rollup/plugin-replace';
 import { execSync } from 'child_process';
 import { terser } from "rollup-plugin-terser";
+import path from 'path';
 
-const handposeWorkerCompileCmd = `./node_modules/.bin/rollup -f esm -p "rollup-plugin-terser" ./build/esm/handpose.js`;
+const handposeWorkerCompileCmd = path.resolve(`./node_modules/.bin/rollup -f esm -p "rollup-plugin-terser" ./build/esm/handpose.js`);
```

Comment:
This seems odd to me that it works on the full command. That doesn't seem to be a supported use case from the documentation.
https://nodejs.org/api/path.html#pathresolvepaths
I would use the path.resolve function separately on ./node_modules/.bin/rollup and ./build/esm/handpose.js
and use a template literal with variables here just to be sure. Can you please do that change?

Reply:
Good catch, I didn't even realize when I was working on it since the build command still finished somehow. Fixed now.